src/App.css

```diff
@@ -1,3 +1,197 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap");
+
 * {
   margin: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+:root {
+  --black: #121212;
+}
+
+body {
+  background-color: #f5f6fa;
+}
+
+.navBar {
+  background-color: white;
+  box-shadow: 10px 5px 5px rgba(0, 0, 0, 0.25);
+  align-items: center;
+  height: 10vh;
+  width: 100%;
+}
+
+.navList {
+  width: 70%;
+  margin-left: 6.5rem;
+  align-items: center;
+  justify-content: space-evenly;
+  list-style: none;
+}
+
+link {
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 1.874rem;
+  color: #0290ff;
+  font-weight: bold;
+}
+
+.book {
+  width: 75rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 0.7rem 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.bookInfo {
+  display: flex;
+  flex-direction: column;
+  margin-left: 15px;
+  align-items: flex-start;
+  row-gap: 5px;
+  width: 30%;
+}
+
+.category {
+  color: var(--black);
+  font-weight: bold;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.title {
+  color: var(--black);
+  font-size: 1.375rem;
+}
+
+.author {
+  color: #4386bf;
+  font-weight: 300;
+  font-size: 0.875rem;
+}
+
+.actions {
+  display: flex;
```

Comment:
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
### _This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)